assets/src/design-system/components/input/index.js

```diff
@@ -174,6 +174,10 @@ export const Input = forwardRef(
     if (unit && value.length) {
       displayedValue = `${value}${!focused ? `${unit}` : ''}`;
     }
+    if (isIndeterminate) {
+      // Display placeholder if value couldn't be determined.
+      displayedValue = null;
```

Comment:
(This is not actually working currently since it doesn't seem to rerender when adding another element to the selection.)